.github/workflows/healthchecks_azurekeyvault_cd_preview.yml

```
@@ -8,23 +8,9 @@ on:
 
 jobs:
   build:
-    env:
+    uses: ./.github/workflows/reusable_cd_preview_workflow.yml
```

```suggestion
    uses: ./.github/workflows/reusable_cd_preview_workflow.yml
    secrets: inherit
```
We will need this so we can use ${{secrets.NUGET_API_KEY}}